POLMED/src/components/footers/style.scss

```diff
@@ -0,0 +1,13 @@
+@import "../../styles/global-styles";
+
+.footer {
+  @include flex(column);
+  background: linear-gradient(100deg, var(--primary), var(--lime-green));
+  padding: 1em 0;
+  width: 100%;
+}
+
+.hp-footer-lower {
```

Comment:
fajne rozwiązanie z przeniesieniem wspólnych styli dla obu footerów do jednego pliku.
skoro w klasie footer usunales "hp" - to zrob to samo dla footer-lower, tak aby te style wspolne mialy taka sama konwencje nazywania - bez przedrostka "hp"

Reply:
ewentualnie jeśli font-size w przypadku hp ma być inny niż w przypadku reszty, to rozdziel te style do dwoch osobnych plikow - jeden styl w tym, a drugi w tamtym co ja robilem

Reply:
ale to juz sam zdecyduj font-size moze byc dla obu taki sam - sam przeciez projektowales stronke :D